src/app/app.component.scss

```diff
@@ -16,6 +16,7 @@
   .card-band {
     grid-area: card-band;
     border-bottom: solid 0.5px #cccccc;
+    display: flex;
```

Comment:
Is this needed? Wouldn't it be better to set `display: flex` on the root of `card-deck`?

Reply:
yep, but that's still placeholder styling. removed anyway.